lib/remote_persistent_term/fetcher/s3.ex

```diff
@@ -35,6 +44,26 @@ defmodule RemotePersistentTerm.Fetcher.S3 do
       type: {:in, [:gzip]},
       required: false,
       doc: "The compression algorithm used to compress the remote term."
+    ],
+    retry: [
+      type: :keyword_list,
+      required: false,
+      default: [],
+      doc: "Options for retry behavior when fetching fails.",
+      keys: [
+        failover_region: [
```

Comment:
i was thinking this should be a list, in case you have more than one fail over region

Reply:
ah yeah cool idea